doc/man3/ERR_put_error.pod

```diff
@@ -4,7 +4,8 @@
 
 =head1 NAME
 
-ERR_raise, ERR_raise_data,
+ERR_raise, ERR_raise_data, CRYPTO_malloc_with_err,
+CRYPTO_realloc_with_err, CRYPTO_clear_realloc_with_err,
```

Comment:
Could you please move the documentation into the OPENSSL_malloc.pod instead? It seems to be more appropriate to put it there given the prefix and the functionality.

Reply:
Makes much sense. Done.